spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/HealthEndpointWebExtension.java

```diff
@@ -76,6 +81,10 @@ public HealthEndpointWebExtension(HealthContributorRegistry registry, HealthEndp
 		}
 		HealthComponent health = result.getHealth();
 		HealthEndpointGroup group = result.getGroup();
+		if (!Status.UP.equals(health.getStatus())) {
+			logger.debug("health status=" + health.getStatus().getCode() + " with description="
```

Comment:
Should this be debug or warn ? Since the app is not un up state, warn sounds more logical to me

Reply:
I agree, that `warn` would be the preferred level, but I didn't want to change the default behavior of Spring Boot for all Spring Boot users. In addition the log level will be `warn` in #22632 I guess. Thus its just additional logging for those who want to have it (by setting the class-level to `debug`). I also hope it willl make it earlier to `master` than #22632.

Reply:
>  I also hope it willl make it earlier to master than #22632.

Sorry but it won't. This issue or the other one is going to be superseded and I already have some work on a branch when I was working on #22632.